src/main/paradox/docs/additional-info/faq.md

```diff
@@ -1,3 +1,124 @@
+**How can I try Nexus without installing it? What is the difference with a regular database like PostgreSQL?**
+
+
+
+**Is there a cloud deployment of Nexus?**
+
+There is a sandbox for you to try Nexus, it has limited resources and is regularly wiped out. Further information [here](https://bluebrain.github.io/nexus/docs/webapps/index.html)
+
+
+**Is there a limit on the number of resources Nexus can store?**
+
+
+
+**What is a Knowledge Graph?**
+
+Knowledge Graph is an innovative tool that Google launched in May 2012 to enhance its search engine results, gathering informations from a variety of sources. It serves as data integration hub to connect data by their semantic meaning, in the form of the ontology, and organise them as a graph, allowing a flexible formal data structure.
+At the heart of Blue Brain Nexus platform lies Knowledge Graph, that provide knowledge representation to enable FAIR principles at Blue Brain and in neuroscience community.
+Indeed, Knowledge Graph Nexus allow scientists to: 1. register and manage neuroscience relevant entity types; 2. Submit data to the platform and describe their provenance using the W3C PROV model; 3. Search, discover, reuse and derive high-quality neuroscience data generated within and outside the platform for the purpose of driving their own scientific endeavours.
+It is explained in 'Understanding Knowledge Graph' [page](https://bluebrain.github.io/nexus/docs/tutorial/knowledge-graph/index.html).
+
+
+**How Nexus support data governance?**
+
+
+**What has to be configured and how an instance can be integrated with Nexus v1?**
+
+
+**How do I report a bug? What information should I include?**
+
+If you have found a bug while using some of the Nexus services, please create an issue labelled “bug” in the BlueBrain/nexus repo on GitHub [here](https://github.com/BlueBrain/nexus/issues/new?labels=bug). You have to include all the details about your case.
+
+
+**Which support Nexus team provide?**
+
+
+
+## Technical
+
+**How can I configure my project?**
+
+**How to ingest my data into Nexus**
+
+(csv, json, json-ld, no schema needed)
+
+**I uploaded my data, but the system is not working. What can I do?**
+
+If ‘Replicas’ is 0/1 there’s a good chance that docker is still pulling the data.
+Then, if you run docker stack ps nexus you can see if everything is up.
+
+
+**What are the clients I can use with Nexus? What are the requirements to run Nexus locally?**
+
+On macOS and Windows, Docker effectively runs containers inside a VM created by the system hypervisor. Nexus requires at least 2 CPUs and 8 GiB of memory in total. You can increase the limits in Docker settings in the menu *Preferences > Advanced*. Further information [here](https://bluebrain.github.io/nexus/docs/getting-started/running-nexus/docker.html).
+
+
+**Where is the sandbox deployed?**
+
+
+**What is a SHACL schema?**
+
+
+
+**How does SHACL compares with JSON Schema?**
+
+
+
+**Do I need to define SHACL schemas to bring data in?**
+
```

Comment:
No. SHACL schemas provides with an extra layer of quality control for the data that is ingested into Nexus. However we acknowledge the complexity of defining schemas. That's why clients can decide whether to use schemas to constrain their data or not, depending on their use case and their available resources.